cpp/Makefile

```diff
@@ -71,7 +71,7 @@ test-examples-host:
 
 .PHONY: ci-image
 ci-image: hdoc-build
-	docker build -t mcap_cpp_ci -f ci.Dockerfile --build-arg IMAGE=ubuntu:focal --build-arg CC=clang-13 --build-arg CXX=clang++-13 .
+	docker build -t mcap_cpp_ci -f ci.Dockerfile --build-arg IMAGE=ubuntu:jammy --build-arg CC=clang-13 --build-arg CXX=clang++-13 .
```

Comment:
using latest ubuntu to get latest cmake version needed to create an interface library